util/syslog.py

```diff
@@ -5,7 +5,7 @@
 from .configuration import Messaging
 from discord import Webhook, RequestsWebhookAdapter
 
-sys.excepthook = lambda type, value, tb: handleUncaughtException(type, value, tb)
+sys.excepthook = lambda type, value, tb: __handleUncaughtException(type, value, tb)
```

Comment:
Why the "__"? Just because of Python syntax?

Reply:
Yeah, because I realized that the `handleUncaughtException` method was a private method, and therefore didn't need a doc string. So I changed the method to "private" designated.

Reply:
Ah, fair enough. Thanks for the info. :)